src/include/mallocMC/distributionPolicies/XMallocSIMD_impl.hpp

```diff
@@ -33,10 +33,12 @@
 
 #pragma once
 
-#include "../mallocMC_prefixes.hpp"
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
I had a hard time getting XMallocSIMD working on CPU platforms, so I eventually just disabled it. Is there even a point of having it available on CPUs?